src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import React, { useState } from 'react';
+
+import { TodoList } from './components/TodoList';
+import { User } from './components/UserInfo';
+
+export type Todo = {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+  user: User | null;
+};
+
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
```

Comment:
[CODE KNOWLEDGE] - When you have utility functions like `getUserById`, consider moving them to a separate utilities or services file. This will keep your main App component cleaner and your utility functions will be more reusable and easier to manage.